src/Font.cpp

```diff
@@ -14,7 +14,7 @@
 #include "File.h"
 
 // Uncomment for that authentic pre-Windows Vista feel
-//#define DISABLE_FONT_ANTIALIASING
+// #define DISABLE_FONT_ANTIALIASING
```

Comment:
```suggestion
//#define DISABLE_FONT_ANTIALIASING
```
I use '// ' for comments, but not commented-out code.